activitysynth/scripts/datasources.py

```diff
@@ -192,6 +206,55 @@ def skims(data_mode, store, s3_input_data_url, local_data_dir, csv_fnames):
     return df
 
 
+# BEAM Skims
+@orca.table()
+def beam_drive_skims(
+        data_mode, store, s3_input_data_url, local_data_dir, csv_fnames):
+    """
+    Load BEAM skims, convert travel time to minutes
+    """
+    if data_mode == 's3':
+        df = pd.read_parquet(s3_input_data_url.format('beam_drive_skims'))
+    elif data_mode == 'h5':
+        df = store['beam_drive_skims']
+    elif data_mode == 'csv':
+        df = pd.read_csv(local_data_dir + csv_fnames['beam_drive_skims'])
+
+    # morning peak
+    df = df[df['period'] == 'AM']
```

Comment:
so if we ever want the offpeak and PM skims, we would need to add those as separate beam_drive_skims_[period] tables, and also add an offpeak_lookup / pm_lookup function in the models.py impute_missing_skims function below?

Reply:
Yeah, I mean, we could add it in here in this PR if you need it now. I just wanted to get something working checked in that we could each tweak to meet our individual needs. If you need PM just get started though I can add it?